src/components/preview/preview.component.tsx

```diff
@@ -1,40 +1,63 @@
 import React from "react";
 import { MarginProps } from "styled-system";
-
-import PreviewPlaceholder, {
-  PreviewPlaceholderProps,
-} from "./__internal__/preview-placeholder.component";
-import { StyledPreview } from "./preview.style";
+import { StyledPreview, StyledPreviewPlaceholder } from "./preview.style";
 import { filterStyledSystemMarginProps } from "../../style/utils";
+import useMediaQuery from "../../hooks/useMediaQuery";
+
+export type Shapes = "text" | "rectangle" | "rectangle-round" | "circle";
 
-export interface PreviewProps
-  extends Partial<Omit<PreviewPlaceholderProps, "index">>,
-    MarginProps {
+export interface PreviewProps extends MarginProps {
   /** Children content to render in the component. */
   children?: React.ReactNode;
-  /** Provides more control over when in a loading state. */
+  /** Sets loading state. */
   loading?: boolean;
+  /** Sets the height the Preview. */
+  height?: string;
+  /** Sets the width of the Preview. */
+  width?: string;
+  /** The number of Previews to render. */
+  lines?: number;
+  /** Sets the preview's shape. */
+  shape?: Shapes;
+  /** Removes Preview's animation, is true when prefer reduce-motion is on. */
+  isStatic?: boolean;
 }
 
 export const Preview = ({
   children,
   loading,
   lines = 1,
+  height,
+  width,
+  shape = "text",
+  isStatic,
```

Comment:
comment: I notice we have a similar prop [here](https://github.com/Sage/carbon/blob/2783aceed1b13b5bdaee9cc7ec87ab8fea118b94/src/components/popover-container/popover-container.component.tsx#L183), I think it might be worth using the same name in this component for consistency